### PR TITLE
Saskatchewan Active families benefit (AFB)

### LINKED ISSUE #206: Saskatchewan Active Families Benefit
[Reference ](https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/saskatchewan/5008-pc/information-residents-saskatchewan.html#AFB)

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #206

### CODE REVIEW

File: policyengine_canada/parameters/gov/provinces/sk/benefits/afb/base.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Sasktachewan Active Family Benefit child base.
+values: 
+  2021-01-01: 150
+metadata:
+  unit: currency-CAD
+  label: Sasktachewan Active Family Benefit Child Base
+  reference: 
+  - title: Sasktachewan Active Families Benefit
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/sk/benefits/afb/base.yaml:8]: Any luck finding a legal reference? If not could you add this comment.
```suggestion
  # Can't find a legal reference.
  - title: Sasktachewan Active Families Benefit
```

File: policyengine_canada/variables/gov/provinces/sk/benefits/afb/sk_active_family_benefit.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class sk_active_family_benefit(Variable):
+    value_type = float
+    entity = Household
+    label = "Sasktachewan Active Family Benefit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.SK
+
+    def formula(household, period, parameters):
+        income = household("adjusted_family_net_income", period)
+        children_nondisabled = household("sk_afb_eligible_children", period)
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/sk/benefits/afb/sk_active_family_benefit.py:13]: I don't think we need so many variables:
```suggestion
        p = parameters(period).gov.provinces.sk.benefits.afb
        person = household.members
        child = person("age", period) <= p.age_eligibility
        disabled = person("is_disabled", period)
        children = household.sum(child)
        disabled_children = household.sum(child & disabled)
        non_disabled_children = children - disabled_children
        eligible = income <= p.income_threshold
        amount_if_eligible = (p.amount.non_disabled * non_disabled_children) + (p.amount.disabled * disabled_children)
        return eligible * amount_if_eligible
```

File: policyengine_canada/parameters/gov/provinces/sk/benefits/afb/base.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/sk/benefits/afb/base.yaml:1]: Please reorganize this and disabled_base into `amount/{disabled,non_disabled}.yaml`

File: policyengine_canada/parameters/gov/provinces/sk/benefits/afb/age_eligibility.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/sk/benefits/afb/age_eligibility.yaml:1]: Reorganize into `eligibility/{age,income}.yaml`

**Comment by MaxGhenis** [comment]
also `make format`

**Comment by PavelMakarchuk** [comment]
> also `make format`

done

**Pull request merged**